lua/telescope/actions/init.lua

```diff
@@ -546,10 +546,19 @@ actions.git_staging_toggle = function(prompt_bufnr)
   end
 end
 
+local make_entry_fname_abs = function(entry)
```

Comment:
Only comment here is that usually the prefixes for `make_entry_*` functions are the actual `entry_maker` functions themselves, so it might be confusing as this function isn't actually an `entry_maker` per se, but rather altering a attribute of an entry.

Also this might go better in `util.lua` or similar rather than here where we are declaring baseline actions?

Reply:
Actually that second comment about putting this in `util.lua` might not make sense, as the `entry` argument is contextual in that it must be a table with `cwd` and `filename` keys, etc.